### PR TITLE
Efficiency docs

### PR DESCRIPTION
**Pull request by kwombach12**
## Proposed Changes

Adding Documentation for the new Efficiency Dashboard

### CODE REVIEW

File: images/Efficiency-dashboard.png
Comment (chipzoller) [inline images/Efficiency-dashboard.png:1]: A good practice when dealing with images which appear on one page is to name them all using lower case, beginning with the same string. This way they're all grouped together making it much more clear, if we need to do site migrations or maintenance, that they all belong to the same page. So, I suggest making all these images start with "efficiency" which means updating the links in the Markdown.

File: using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+- _**Workload Idle**_- Workload Idle is defined as the cost of resources which are requested, but not used, by Kubernetes workloads.
+For example, in a pod with a single container which requests 2 Gi of memory but only uses 1 Gi, there is 1 Gi of workload idle which is assigned a cost. **This is a new definition of ‘idle’ for Kubecost.**
+- _**Infra Idle**_ - Infra Idle (Infrastructure Idle) is defined as the difference between the cost of allocated resources and the cost of the hardware on which they run. Allocation is defined as the max of usage and request.
+For example, a Kubernetes node which has 32 Gi of usable memory runs two pods each with one container. The first pod requests 2 Gi of memory but only uses 1 Gi. The second pod specifies no requests but uses 3 Gi. There is 27 Gi of Infra Idle which is assigned a cost. This is the same definition of ‘idle’ as used in the Kubecost Allocations page (see more details [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/cost-allocation/efficiency-idle#idle)).
```
Comment (chipzoller) [inline using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md:19]: When you reference other locations in the docs, don't use a static link like this, use a relative link. Reason being is so links don't become brittle and makes the content portable.

File: using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
+- _**Total Idle**_ - The sum of Infra Idle and Workload Idle. 
+- _**Cluster Efficiency**_ - The ratio of the cost of used resources to the total cost of all resources. (cost of resources used) / (total cost of resources). This is the efficiency metric presented in the upper right-hand corner of the Overview page.
+
+![Cluster Efficiency on Overview](/images/cluster-efficiency-on-overview.png)
+
+- _**Workload Efficiency**_ - The cost-weighted ratio of resources used to resources requested. Workload Efficiency is used in the Allocations Dashboard as well (see more details [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/cost-allocation)).
```
Comment (chipzoller) [inline using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md:25]: Ditto on this link.

File: using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md [hunk 3]
```diff
@@ -31,24 +38,27 @@ We can also observe that the workload efficiency in this example would be $50 (c
...
 It is important to remember that workload idle is defined as the cost of resources which are requested but not used. There may be cases where a given workload’s usage exceeds its request – for these cases, the workload idle costs will be $0.00. 
 
 **How can I reduce my workload idle costs?**
 Kubecost is full of insights and automation to help you reduce your wasted workload idle cost! 
-Specifically, the right-size container requests page can help drive significant reductions in workload idle. 
-Additionally, the request sizing Kubecost Action can help you automatically resize your container requests to significantly reduce workload idle. 
-Finally, we recommend reviewing our abandoned workloads page and cleaning up workloads that are no longer needed to reduce workload idle waste. 
+Specifically, the [*right-size container requests*](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/container-request-right-sizing-recommendations) page can help drive significant reductions in workload idle.
```
Comment (chipzoller) [inline using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md:54]: Don't use bold or italics when making links as it's a Markdown no-no.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Looks good, just a few best practice things to tidy up!

**Comment by chipzoller** [comment]
When you're ready, @kwombach12, please re-request review:
image

**Comment by kwombach12** [comment]
@chipzoller I believe I've addressed your comments.

**Review by chipzoller — APPROVED** [review]
Nice work!

**Pull request merged**